presto-docs/src/main/sphinx/presto_cpp/properties.rst

```diff
@@ -0,0 +1,140 @@
+===============================
+Presto C++ Properties Reference
+===============================
+
+This section describes Presto C++ configuration properties.
+
+The following is not a complete list of all configuration and
+session properties, and does not include any connector-specific
+catalog configuration properties. For information on catalog 
+configuration properties, see :doc:`Connectors </connector/>`.
+
+.. contents::
+    :local:
+    :backlinks: none
+    :depth: 1
+
+Coordinator Properties
+----------------------
+
+Set the following properties in the Presto coordinator to enable its use of Presto C++ workers. 
+
+.. code-block:: none
+
+    native-execution-enabled=true
+    optimizer.optimize-hash-generation=false
+    regex-library=RE2J
+    use-alternative-function-signatures=true
+    experimental.table-writer-merge-operator-enabled=false
+
+``experimental.table-writer-merge-operator-enabled``
```

Comment:
This also has session property available `table_writer_merge_operator_enabled`, should we document that as well here? 
https://github.com/prestodb/presto/blob/master/presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java#L1146

Reply:
I don't think `experimental.table-writer-merge-operator-enabled` is supported. Hence it has to be set to `false` explicitly. `true` is not an option. We have to clarify that below.